src/strategies/chubbykaijudao/examples.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "ChubbyKaijuDAO Stakers and Holders",
+    "strategy": {
+      "name": "chubbykaijudao-stakers-and-holders",
```

Comment:
Strategy name should be same as folder name @ekwelfweer Change the name here or change the folder name

Reply:
yep. i changed it all to "chubbykaijudao"